src/main/kotlin/com/hedvig/underwriter/model/QuoteDao.kt

```diff
@@ -77,6 +78,24 @@ interface QuoteDao {
     )
     fun find(@Bind quoteId: UUID): DatabaseQuoteRevision?
 
+    @SqlQuery(
+        """
+            SELECT
+            DISTINCT ON (qr.master_quote_id)
+
+            qr.*,
+            mq.created_at,
+            mq.initiated_from
+
+            FROM quote_revisions qr
+            INNER JOIN master_quotes mq
+                ON mq.id = qr.master_quote_id
+            WHERE qr.master_quote_id in (<quoteIds>)
+            ORDER BY qr.master_quote_id ASC, qr.id DESC
+        """
+    )
+    fun find(@BindList("quoteIds") quoteIds: List<UUID>): List<DatabaseQuoteRevision?>
```

Comment:
```suggestion
    fun find(@BindList("quoteIds") quoteIds: List<UUID>): List<DatabaseQuoteRevision>
```

Why we need to be nullable here? If the query doesnt find anything, it will return an empty list right?